src/Makefile.am

```diff
@@ -1,6 +1,6 @@
 DIST_SUBDIRS = secp256k1 univalue
 
-AM_LDFLAGS = $(PTHREAD_CFLAGS) $(LIBTOOL_LDFLAGS) $(HARDENED_LDFLAGS)
+AM_LDFLAGS = $(PTHREAD_CFLAGS) $(LIBTOOL_LDFLAGS) $(HARDENED_LDFLAGS) -L/usr/lib/x86_64-linux-gnu -L/usr/local/cuda-9.2/lib64
```

Comment:
This creates linker issues on non-linux operating systems such as OSX.

Reply:
It just wasn't supposed to be there ^^